test/README.md

```diff
@@ -0,0 +1,15 @@
+# Running tests
+
+This folder has all the system test files for lxd.
+
+## Dependencies
+
+You'll need the following dependencies installed before you can run them:
+
+    sudo apt-get install jq sqlite3 golang-go
+
+
+## Running
+
+A simple "cd test && sudo ./main.sh" should be all you need to run the system
```

Comment:
On Mon, May 04, 2015 at 07:23:02AM -0700, Chris Glass wrote:
> > @@ -0,0 +1,15 @@
> > +# Running tests
> > +
> > +This folder has all the system test files for lxd.
> > +
> > +## Dependencies
> > +
> > +You'll need the following dependencies installed before you can run them:
> > +
> > +    sudo apt-get install jq sqlite3 golang-go
> > +
> > +
> > +## Running
> > +
> > +A simple "cd test && sudo ./main.sh" should be all you need to run the system
> 
> Nice, that's a good trick. How about adding a Makefile target for this to make sure it's not forgotten? something like "make test"? (I volunteer for sending a PR if that is deemed valuable).

We have make check; could we just add it there? sudo -E ./main.sh
insead of just calling main.sh so you don't have to run the whole
thing as root?

> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/lxc/lxd/pull/582/files#r29587945